tests/test_internals/test_tools.py

```diff
@@ -76,3 +76,37 @@ def test_get_globus_dataset_content_sizes():
         "YutaMouse41-150821.temp.clu.7": 214835,
         "YutaMouse41-150821.temp.clu.8": 174434,
     }
+
+
+def test_get_s3_conversion_cost_standard():
+    test_sizes = [
+        1,
+        100,
+        1e3,  # 1 GB
+        1e5,  # 100 GB
+        1e6,  # 1 TB
+        1e7,  # 10 TB
+        1e8,  # 100 TB
+    ]
+    results = [get_s3_conversion_cost(total_mb=total_mb) for total_mb in test_sizes]
+    assert results == [
+        2.9730398740210563e-15,  # 1 MB
+        2.973039874021056e-11,  # 100 MB
+        2.9730398740210564e-09,  # 1 GB
+        2.9730398740210563e-05,  # 100 GB
+        0.002973039874021056,  # 1 TB
+        0.2973039874021056,  # 10 TB
+        29.73039874021056,  # 100 TB
+    ]
+
+
+@pytest.mark.skipif(
+    not HAVE_GLOBUS or not LOGGED_INTO_GLOBUS,
+    reason="You must have globus installed and be logged in to run this test!",
+)
+def test_get_s3_conversion_cost_from_globus():
+    content_sizes = get_globus_dataset_content_sizes(
+        globus_endpoint_id="188a6110-96db-11eb-b7a9-f57b2d55370d",
+        path="/SenzaiY/YutaMouse41/YutaMouse41-150821/originalClu/",
+    )
+    assert get_s3_conversion_cost(total_mb=sum(content_sizes.values()) / 1e6) == 1.756555806400279e-13
```

Comment:
This is not a linear equation, so sum(get_s3_conversion_cost(x) for x in sizes) != get_s3_conversion_cost(sum(sizes)). This works here because it's one session, but for multiple sessions you would need to do sum(get_s3_conversion_cost(x) for x in sizes)

Reply:
Ah, yeah my bad

Reply:
Right, the contents above show it's a collection of Neuroscope spike sorting files (one per shank) so I'm estimate the cost of a single NeuroscopeSorting conversion there.

But yes, it is important to remember that when calculating total cost of a dataset over multiple sessions, you'd have to get the cost for each session and then add all the costs.